arch/sim/src/Makefile

```diff
@@ -71,10 +71,16 @@ CSRCS += up_unblocktask.c up_blocktask.c up_releasepending.c
 CSRCS += up_reprioritizertr.c up_exit.c up_schedulesigaction.c
 CSRCS += up_allocateheap.c
 
+ifeq ($(CONFIG_SIM_PREEMPTIBLE),y)
+  CSRCS += up_tick_me.c
+  STDLIBS += -lpthread
+endif
+
 VPATH = sim
 DEPPATH = $(patsubst %,--dep-path %,$(subst :, ,$(VPATH)))
 
 HOSTSRCS = up_hosttime.c
+HOSTCFLAGS += -DCONFIG_SIM_PREEMPTIBLE_TICK_DURATION_MS=$(CONFIG_SIM_PREEMPTIBLE_TICK_DURATION_MS)
```

Comment:
I suggested to use USEC_PER_TICK before.

Reply:
I will update my patch